app/helpers/navigation_items.rb

```diff
@@ -17,7 +17,7 @@ def for_candidate_interface(current_candidate, _current_controller)
     end
 
     def for_support_primary_nav(current_support_user, current_controller)
-      if current_support_user
+      if current_support_user && !is_active(current_controller, %w[performance_dashboard])
```

Comment:
Let's drop this conditional, it's not that important that there's no nav for this page since it's not displayed on TVs anymore (the nav is even quite useful).